book/RayTracingGPUEdition.html

```diff
@@ -4904,11 +4904,9 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Rust
     ...
```

Comment:
Let's add the function signature after the first ellipsis to stay consistent with other listings, like so:

```suggestion
    ...
    async fn main() -> Result<()> {
```